PCGen-Formula/build.gradle

```diff
@@ -45,6 +45,12 @@ repositories {
 }
 
 dependencies {
+    // Use this if you want to reference your own local pcgen base, alter the 
+    // relative path to what you need.
+    //compile files("../../pcgen-base/PCGen-base/build/libs/PCgen-base-1.0.jar")
+    
+    // TODO We'll need to build a 1.0.135 version that contains the Java 11 
```

Comment:
I don't understand.  There is a VM that builds and stores the libraries that I maintain, since the main system is not building correctly